### PR TITLE
Improvment/csi 3382 topology remove general topology key

### PR DESCRIPTION
**Pull request by ArbelNathan**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ var (
 	scheme               = runtime.NewScheme()
 	setupLog             = ctrl.Log.WithName("setup")
 	watchNamespaceEnvVar = "WATCH_NAMESPACE"
-	topologyPrefixes     = [...]string{"topology.kubernetes.io", "topology.block.csi.ibm.com"}
+	topologyPrefix       = "topology.block.csi.ibm.com"
```
Comment (oriyarde) [inline main.go:47]: it can still be a list, with one item

Comment (ArbelNathan) [reply main.go:47]: for what purpose?

Comment (oriyarde) [reply main.go:47]: so it would be easy to add/remove prefixes

Comment (ArbelNathan) [reply main.go:47]: do we have any intention of doing it?

Comment (oriyarde) [reply main.go:47]: theoretically in the future (and definitely in this PR), sure

Comment (ArbelNathan) [reply main.go]: ok, done

**Review by oriyarde — APPROVED** [review]
LGTM

**Pull request merged**